### PR TITLE
Removes misc compiler warnings.

### PR DESCRIPTION
**Pull request by StephenHodgson**
Assigned default values, if none.
Updated `WebRequest.send()`.
Updated null check because structs can't be null.

### CODE REVIEW

File: Assets/HoloToolkit/Utilities/Scripts/GLTF/Scripts/GLTFComponent.cs [hunk 1]
```diff
@@ -6,13 +6,13 @@ namespace GLTF
...
     {
-        public string Url;
+        public string Url = string.Empty;
         public bool Multithreaded = true;
 
         public int MaximumLod = 300;
 
-        public Shader GLTFStandard;
-        public Shader GLTFConstant;
+        public Shader GLTFStandard = null;
```
Comment (keveleigh) [inline Assets/HoloToolkit/Utilities/Scripts/GLTF/Scripts/GLTFComponent.cs]: Do these not default to null?

File: Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs [hunk 2]
```diff
@@ -121,17 +121,20 @@ private void RenderFloorQuad()
...
+                        if (dimensions != Vector3.zero)
                         {
-                            Debug.Log("Drawing floor at dimensions Y.");
-                            // Draw the floor at boundary Y.
+                            Debug.Log("Drawing floor at height offset: " + dimensions.y);
                             floorQuadInstance.transform.localPosition = new Vector3(0, dimensions.y, 0);
                         }
+                        else
+                        {
+                            Debug.Log("Drawing floor at 0,0,0.");
```
Comment (keveleigh) [inline Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs]: This if/else is basically "if dimensions is Vector3.zero, draw the floor at Vector3.zero". If dimensions can never be null, we can probably remove the check entirely and just draw at Vector3(0, dimensions.y, 0)

File: Assets/HoloToolkit/Utilities/Scripts/GLTF/Scripts/GLTFComponent.cs (hunk 1 above)
Comment (StephenHodgson) [reply Assets/HoloToolkit/Utilities/Scripts/GLTF/Scripts/GLTFComponent.cs]: They do, but I like a clean console, that doesn't have warnings.

**Comment by DDReaper** [comment]
All errors in the fixed scripts have gone.  Only script warnings left are : 

- "UNetAnchorManager.cs" (assigned but not used warnings)
- "ControllerInfo.cs" (Lots of assigned but not used warnings)

**Comment by StephenHodgson** [comment]
@DDReaper I'll be addressing those two issues in future PRs related to those specific feature areas.

**Comment by DDReaper** [comment]
"UNetAnchorManager.cs" fixed in #1024

**Pull request merged**